src/Configuration/src/SpringBoot/SpringBootCommandLineProvider.cs

```diff
@@ -7,7 +7,7 @@
 namespace Steeltoe.Configuration.SpringBoot;
 
 /// <summary>
-/// Configuration provider that expands '.' delimited configuration keys that start with "spring." (typically originating from Spring Boot command-line
+/// Configuration provider that expands '.'-separated configuration keys that start with "spring." (typically originating from Spring Boot command-line
```

Comment:
```suggestion
/// Configuration provider that expands '.' separated configuration keys that start with "spring." (typically originating from Spring Boot command-line
```
This looks weird... I think not using the dash, `period-separated` or `dot-separated` would maybe be better? same in the next file

Reply:
Or "dot notation" https://builtin.com/data-science/dot-notation

Reply:
Pushed changes.